modules/mobile/app/controllers/mobile/v0/letters_controller.rb

```diff
@@ -109,13 +109,16 @@ def increment_download_counter
       # body params appear in the params hash in specs but not in actual requests
       def download_options_hash
         body_string = request.body.string
```

Comment:
The logging I had only printed the body_string and not the other objects so I've broken it up into more distinct logging. Will remove later.